src/main/resources/application.properties

```diff
@@ -1,6 +1,7 @@
 spring.application.name=holidays
 
 spring.h2.console.enabled=true
+spring.h2.console.settings.web-allow-others=true
```

Comment:
This is to handle error "sorry remote connections ('weballowothers') are disabled on this server" while running spring boot app as docker container